tox.ini

```diff
@@ -22,10 +22,10 @@ passenv = DJANGO_SETTINGS_MODULE PYTHONPATH HOME DISPLAY OBJC_DISABLE_INITIALIZE
 setenv =
     PYTHONDONTWRITEBYTECODE=1
 deps =
-    py{3,36,37,38}: -rtests/requirements/py3.txt
-    postgres: -rtests/requirements/postgres.txt
-    mysql: -rtests/requirements/mysql.txt
-    oracle: -rtests/requirements/oracle.txt
+    py{3,36,37,38}: --editable .[testing]
+    postgres: --editable .[postgresql]
+    mysql: --editable .[mysql]
+    oracle: --editable .[oracle]
```

Comment:
This can use the [tox extras](https://tox.readthedocs.io/en/latest/config.html#conf-extras) instead of `deps`.

```
extras:
    testing
    postrgres: postgresql
    ...
```